src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/FtpUploadTest.java

```diff
@@ -24,7 +24,10 @@ void uploads_file() throws Exception {
         FileToSend doc = new FileToSend("hello.zip", "world".getBytes(), false);
         try (LocalSftpServer server = LocalSftpServer.create()) {
             FtpClient client = FtpHelper.getSuccessfulClient(LocalSftpServer.port);
-            client.upload(doc, LocalSftpServer.SERVICE_FOLDER);
+            client.runWith(sftpClient -> {
```

Comment:
I would suggest to rename FtpClient -> FtpHelper or FtpUtil. Otherwise it is causing confusion while reading the code looks like FTP client run with take sftp client

Reply:
Some of those tests are cramming actual integration testing purpose - they are there because they are there ¯\\\_(ツ)_/¯. I wouldn't want to move/rename them in this scope as it was already a fundamental change. Since not breaking - best to leave as a separate stand-alone review, sounds good?